app/controllers/welcome_controller.rb

```diff
@@ -13,13 +13,16 @@ def index
 
   def search
     search = params.require(:search).permit(:query, :platform)
-    if search[:platform] == "vimeo"
+    if search[:platform] == "vimeo" && search[:query].empty? == false
       return redirect_to vimeo_results_path(search[:query])
-    elsif search[:platform] == "instagram"
+    elsif search[:platform] == "instagram" && search[:query].empty? == false
       redirect_to instagram_results_path(search[:query])
-    else
+    elsif search[:platform] == nil
       flash[:error] = "Please select instagram or vimeo."
       redirect_to :back
+    else
+      flash[:error] = "How about search a user name?"
```

Comment:
Can the syntax be changed to match the phrasing on line 21? Maybe something like "Please search for a user name." :)